airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json

```diff
@@ -6,12 +6,12 @@
     "type": "object",
     "properties": {
       "account_id": {
-        "title": "Ad Account ID",
-        "description": "The Facebook Ad account ID to use when pulling data from the Facebook Marketing API. The Ad account ID number is in the account dropdown menu or in your browser's address bar of your <a href=\"https://adsmanager.facebook.com/adsmanager/\">Meta Ads Manager</a>. See the <a href=\"https://www.facebook.com/business/help/1492627900875762\">docs</a> for more information.",
+        "title": "Ad Account ID(s)",
+        "description": "Comma-separated list of the Facebook Ad account IDs. The Ad account ID number is in the account dropdown menu or in your browser's address bar of your <a href=\"https://adsmanager.facebook.com/adsmanager/\">Meta Ads Manager</a>. See the <a href=\"https://www.facebook.com/business/help/1492627900875762\">docs</a> for more information.",
         "order": 0,
-        "pattern": "^[0-9]+$",
-        "pattern_descriptor": "1234567890",
-        "examples": ["111111111111111"],
+        "pattern": "^[0-9]+(,[0-9]+)*$",
+        "pattern_descriptor": "The Ad Account ID must be a number. Separate multiple Ad Account IDs using commas.",
+        "examples": ["111111111111111,22222222222"],
```

Comment:
why don't we `array` use for better UX ?
```json
"type": "array",
"items": {
  "type": "string"
}
```

Reply:
Fixed